src/controllers/articles.controller.js

```diff
@@ -105,9 +116,13 @@ class Articles {
         message: 'That article does not exist!'
       });
     }
+    const article = _.pick(findArticle, ['slug', 'title', 'description', 'body', 'taglist', 'favorited', 'favoritedcount', 'flagged', 'images', 'views']);
+    const readTime = Helper.calculateReadTime(article.body);
```

Comment:
Where are You calling .calculateReadTime from ? I don't see it in the helper middleware

Reply:
The calculateReadTime method is in `./src/helpers/helper.js`